### PR TITLE
Signup: Replace Domain Taken Card with Notice

### LINKED ISSUE #33898: Signup: Domain already taken has blue callout left border that no longer works with same-color background
#### Steps to reproduce
1. Walk through the signup process and enter a domain that you know is already taken
2. The box telling you that the domain is already taken has a 3px wide blue left border to call the user's attention to the issue. (See images, below)

#### What I expected

A clear alert that the domain is already taken.

#### What happened instead

An unclear situation that looks more like a visual goof than a callout.

#### Browser / OS version

Any/All

#### Screenshot / Video
This is the way it USED to look (you will notice that the blue bar stands out and is quite visible):

The way it used to look

This is the way it looks today (note that the callout is the same color as the background and it looks not like a callout but a visual goof): 

image

### PR DESCRIPTION
**Pull request by Aurorum**
#### Changes proposed in this Pull Request

Replaces the card that doesn't really match the background ever since it was switched to the blue colour with the more fitting Notice component, as suggested in the initial issue.

#### Testing instructions

Visit `/start/domains-with-preview` and enter an address of a site that is already taken.

**Before:**
Screenshot 2019-12-12 at 21 35 52

**After:**
Screenshot 2019-11-10 at 12 57 45

Fixes #33898

### CODE REVIEW

File: client/components/domains/domain-search-results/index.jsx [hunk 1]
```diff
@@ -144,20 +137,29 @@ class DomainSearchResults extends React.Component {
 				offer = null;
 			}
 
+			const noticeMessage = translate(
+				'If you already own this domain, you can use it for your WordPress.com site.'
```
Comment (a8ci18n) [inline client/components/domains/domain-search-results/index.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'If you already own this domain you can use it for your WordPress.com site.' )` *ES Score: 8*

**Comment by Aurorum** [comment]
Hmm. I'm not sure how to get

**Comment by bluefuton** [comment]
> Hmm. I'm not sure how to use a variable and a translatable string. Any help for this?

@Aurorum Sure! The i18n-calypso README has some good examples:

https://github.com/Automattic/wp-calypso/blob/master/packages/i18n-calypso/README.md

**Comment by Aurorum** [comment]
Thanks @bluefuton, that's super helpful! 

I've updated the PR so that the Linter should now pass, and that the change should now work properly. :)

**Comment by Aurorum** [comment]
Noted! Would you favour the bottom design in [this](https://github.com/Automattic/wp-calypso/issues/33898#issuecomment-501482165) comment, or would it be better to leave it as it is?

**Comment by klimeryk** [comment]
I don't think there was a consensus on what the design should be - and the discussion is from a few months ago. There's been a few changes and iterations around the signup flow. For example, it looks like the side border is different now:

Screenshot 2019-12-19 at 18 37 51

As such, I'll be marking the issue as closed. Sorry if you wasted some time on the fix here 🙇

**Comment by Aurorum** [comment]
> it looks like the side border is different now

I think that's just dependent on the colour scheme. It currently uses `--color-primary-20`, which is a light grey on Nightfall. :)

**Comment by klimeryk** [comment]
Ah, good point about the color scheme! 🙇

**Pull request closed without merging**